fullstack/lib/startCore.ts

```diff
@@ -0,0 +1,15 @@
+import Core, { GlobalPool } from '@ulixee/hero-core';
+import ShutdownHandler from '@ulixee/commons/lib/ShutdownHandler';
+
+let hasStarted = false;
+
+export default function startCore(): void {
+  if (hasStarted) return;
```

Comment:
StartCore doesn't seem to actually start core...

Reply:
Ha. You're right. I was moving way too fast on that.

Importing fullstack into a test was stopping Jest from closing until I moved this code. I have no idea why.

Reply:
Reverted back to original.